app/addons/documents/index-results/components/results/TableRow.js

```diff
@@ -37,7 +37,8 @@ export default class TableRow extends React.Component {
     const row = this.props.data.selectedFields.map(function (k, i) {
 
       const key = 'tableview-data-cell-' + rowNumber + k + i + el[k];
-      const stringified = typeof el[k] === 'object' ? JSON.stringify(el[k], null, '  ') : el[k];
```

Comment:
This looks good. Can you write a test that proves this works and so we don't ever have a regression on this.